### PR TITLE
pitr: support exchange partition, gcs, and azblob

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/11133
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by joccau — APPROVED** [review]
LGTM

File: br/br-log-command-line.md [hunk 1]
```diff
@@ -286,7 +286,7 @@ Some parameters are described as follows:
 
 - `--dry-run`: run the command but do not really delete the files.
 - `--until`: delete all log backup data before the specified timestamp.
-- `--storage`: the backup storage address. The log backup data can only be stored in shared file systems or Amazon S3. For details, refer to [Amazon S3 storage](/br/backup-storage-S3.md).
+- `--storage`: the backup storage address. Currently, BR supports shared file systems, Amazon S3, GCS, and Azure Blob Storage as storage for log backup. For details, see [Amazon S3 storage](/br/backup-storage-S3.md), [GCS storage](/br/backup-storage-gcs.md), and [Azure blob storage](/br/backup-storage-azblob.md).
```
Comment (TomShawn) [inline br/br-log-command-line.md:289]:
```suggestion
- `--storage`: the backup storage address. Currently, BR supports shared file systems, Amazon S3, GCS, or Azure Blob Storage as the storage for log backup. For details, see [Amazon S3 storage](/br/backup-storage-S3.md), [GCS storage](/br/backup-storage-gcs.md), and [Azure blob storage](/br/backup-storage-azblob.md).
```

File: br/br-log-command-line.md [hunk 2]
```diff
@@ -97,7 +97,7 @@ The example output only shows the common parameters. These parameters are descri
 - `--start-ts`: specify the start timestamp for the log backup. If this is not specified, the backup program uses the current time as `start-ts`.
 - `--pd`: specify the PD address for the backup cluster. BR needs to access PD to start the log backup task.
 - `--ca`, `--cert`, `--key`: specify the mTLS encryption method to communicate with TiKV and PD.
-- `--storage`: specify the backup storage address. Currently, BR only supports shared file systems and Amazon S3 as storage for log backup. For details, see [Amazon S3 storage](/br/backup-storage-S3.md).
+- `--storage`: specify the backup storage address. Currently, BR supports shared file systems, Amazon S3, GCS, and Azure Blob Storage as storage for log backup. For details, see [Amazon S3 storage](/br/backup-storage-S3.md), [GCS storage](/br/backup-storage-gcs.md), and [Azure blob storage](/br/backup-storage-azblob.md).
```
Comment (TomShawn) [inline br/br-log-command-line.md:100]:
```suggestion
- `--storage`: specify the backup storage address. Currently, BR supports shared file systems, Amazon S3, GCS, or Azure Blob Storage as the storage for log backup. For details, see [Amazon S3 storage](/br/backup-storage-S3.md), [GCS storage](/br/backup-storage-gcs.md), and [Azure blob storage](/br/backup-storage-azblob.md).
```

File: br/br-log-command-line.md [hunk 3]
```diff
@@ -326,7 +326,7 @@ Global Flags:
 
 This command only accesses the backup storage, but does not access the TiDB cluster.
 
-The `--storage` parameter is used to specify the backup storage address. The log backup data can only be stored in shared file systems or Amazon S3. For details, refer to [Amazon S3 storage](/br/backup-storage-S3.md).
+The `--storage` parameter is used to specify the backup storage address. Currently, BR supports shared file systems, Amazon S3, GCS, and Azure Blob Storage as storage for log backup. For details, see [Amazon S3 storage](/br/backup-storage-S3.md), [GCS storage](/br/backup-storage-gcs.md), and [Azure blob storage](/br/backup-storage-azblob.md).
```
Comment (TomShawn) [inline br/br-log-command-line.md:329]:
```suggestion
The `--storage` parameter is used to specify the backup storage address. Currently, BR supports shared file systems, Amazon S3, GCS, or Azure Blob Storage as the storage for log backup. For details, see [Amazon S3 storage](/br/backup-storage-S3.md), [GCS storage](/br/backup-storage-gcs.md), and [Azure blob storage](/br/backup-storage-azblob.md).
```

File: br/br-log-command-line.md [hunk 4]
```diff
@@ -369,12 +369,12 @@ Global Flags:
 
 The example output only shows the common parameters. These parameters are described as follows:
 
-- `--full-backup-storage`: the storage address for the snapshot (full) backup. If you need to use PITR, you must specify this parameter and choose the latest snapshot backup before the restoration timestamp. If you only need to restore log backup data, you can omit this parameter. To use Amazon S3 as storage, refer to [Amazon S3 storage](/br/backup-storage-S3.md).
+- `--full-backup-storage`: the storage address for the snapshot (full) backup. If you need to use PITR, you must specify this parameter and choose the latest snapshot backup before the restoration timestamp. If you only need to restore log backup data, you can omit this parameter. Currently, BR supports shared file systems, Amazon S3, GCS, and Azure Blob Storage as storage for log backup. For details, see [Amazon S3 storage](/br/backup-storage-S3.md), [GCS storage](/br/backup-storage-gcs.md), and [Azure blob storage](/br/backup-storage-azblob.md).
```
Comment (TomShawn) [inline br/br-log-command-line.md:372]:
```suggestion
- `--full-backup-storage`: the storage address for the snapshot (full) backup. If you need to use PITR, you must specify this parameter and choose the latest snapshot backup before the restoration timestamp. If you only need to restore log backup data, you can omit this parameter. Currently, BR supports shared file systems, Amazon S3, GCS, or Azure Blob Storage as the storage for log backup. For details, see [Amazon S3 storage](/br/backup-storage-S3.md), [GCS storage](/br/backup-storage-gcs.md), and [Azure blob storage](/br/backup-storage-azblob.md).
```

File: br/br-log-command-line.md [hunk 5]
```diff
@@ -369,12 +369,12 @@ Global Flags:
...
 - `--restored-ts`: the timestamp that you want to restore data to. If this parameter is not specified, BR restores data to the latest timestamp available in the log backup, that is, the checkpoint of the backup data.
 - `--start-ts`: the start timestamp that you want to restore log backup data from. If you only need to restore log backup data and do not need snapshot backup data, you must specify this parameter.
 - `--pd`: the PD address of the restoration cluster.
 - `--ca`, `--cert`, `--key`: specify the mTLS encryption method to communicate with TiKV and PD.
-- `--storage`: the storage address for the log backup. The log backup data can only be stored in shared file systems or Amazon S3. For details, refer to [Amazon S3 storage](/br/backup-storage-S3.md).
+- `--storage`: the storage address for the log backup. Currently, BR supports shared file systems, Amazon S3, GCS, and Azure Blob Storage as storage for log backup. For details, see [Amazon S3 storage](/br/backup-storage-S3.md), [GCS storage](/br/backup-storage-gcs.md), and [Azure blob storage](/br/backup-storage-azblob.md).
```
Comment (TomShawn) [inline br/br-log-command-line.md:377]:
```suggestion
- `--storage`: the storage address for the log backup. Currently, BR supports shared file systems, Amazon S3, GCS, or Azure Blob Storage as the storage for log backup. For details, see [Amazon S3 storage](/br/backup-storage-S3.md), [GCS storage](/br/backup-storage-gcs.md), and [Azure blob storage](/br/backup-storage-azblob.md).
```

**Review by TomShawn — APPROVED** [review]
minor comments
LGTM

**Pull request merged**